Cart-And-Wishlist/wishlist.html

```diff
@@ -0,0 +1,184 @@
+            </div>
+
+            <div class="vertical-card-container flex-column">
+                <div class="vertical-card-product-image">
+                    <img src="/Assests/Hooked-book.png" alt="Book Image" class="product-card-image">
+                </div>
+                <div class="vertical-card-content"> 
+                    <h2>Hooked</h2>
+                    <h3 class="card-subheading">by Nir Eyal</h3>
+                    <p class="card-price-list">
+                        <span>Rs. 249 </span>
+                        <strike>Rs. 500</strike>
+                        <i class="fa-solid fa-heart wishlist-label"></i>
+                    </p> 
+                    <div class="vertical-card-button">
+                        <button class="whishlist-btn">Move To Cart</button>
+                        <button class="whishlist-btn"> Remove From WishList</button>    
+                    </div>
+                </div>
+                <span class="card-close">❌</span>
+            </div>
+
+            <div class="vertical-card-container flex-column">
+                <div class="vertical-card-product-image">
+                    <img src="/Assests/Hooked-book.png" alt="Book Image" class="product-card-image">
+                </div>
+                <div class="vertical-card-content"> 
+                    <h2>Hooked</h2>
+                    <h3 class="card-subheading">by Nir Eyal</h3>
+                    <p class="card-price-list">
+                        <span>Rs. 249 </span>
+                        <strike>Rs. 500</strike>
+                        <i class="fa-solid fa-heart wishlist-label"></i>
+                    </p> 
+                    <div class="vertical-card-button">
+                        <button class="whishlist-btn">Move To Cart</button>
+                        <button class="whishlist-btn"> Remove From WishList</button>    
+                    </div>
+                </div>
+                <span class="card-close">❌</span>
+            </div>
+
+            <div class="vertical-card-container flex-column">
+                <div class="vertical-card-product-image">
+                    <img src="/Assests/Hooked-book.png" alt="Book Image" class="product-card-image">
+                </div>
+                <div class="vertical-card-content"> 
+                    <h2>Hooked</h2>
+                    <h3 class="card-subheading">by Nir Eyal</h3>
+                    <p class="card-price-list">
+                        <span>Rs. 249 </span>
+                        <strike>Rs. 500</strike>
+                        <i class="fa-solid fa-heart wishlist-label"></i>
+                    </p> 
+                    <div class="vertical-card-button">
+                        <button class="whishlist-btn">Move To Cart</button>
+                        <button class="whishlist-btn"> Remove From WishList</button>    
+                    </div>
+                </div>
+                <span class="card-close">❌</span>
```

Comment:
I see you have "remove from wishlist" button as well as the cross sign on the card. Both will serve the same purpose of removing the product from the wishlisted page, so my suggestion would be to keep only one(any) acc. to your personal preference.

Reply:
Thanks for review, Removed the card-close one and implemented below comment too.